<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackerPreferencesDialog</class>
 <widget class="QDialog" name="TrackerPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="color_override_layout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="override_locations_background_color">
       <property name="text">
        <string>Override Locations Background Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="override_items_background_color">
       <property name="text">
        <string>Override Items Background Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pick_items_background_color">
       <property name="text">
        <string>Pick Color</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pick_locations_background_color">
       <property name="text">
        <string>Pick Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="override_statistics_background_color">
       <property name="text">
        <string>Override Statistics Background Color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pick_statistics_background_color">
       <property name="text">
        <string>Pick Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="show_location_logic">
     <property name="text">
      <string>Show Location Logic</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_nonprogress_locations">
     <property name="text">
      <string>Show Non-Progress Locations</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="right_click_to_clear_all">
     <property name="text">
      <string>Right Click to Clear All</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clear_all_includes_dungeon_mail">
     <property name="text">
      <string>Clear All Includes Dungeon Mail</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="tracker_save_path_browse_button">
       <property name="text">
        <string>Save Tracker To File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tracker_load_path_browse_button">
       <property name="text">
        <string>Load Tracker From File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TrackerPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TrackerPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
